src/native/corehost/hostmisc/pal.windows.cpp

```diff
@@ -292,6 +294,14 @@ bool pal::get_default_installation_dir(pal::string_t* recv)
     }
 
     append_path(recv, _X("dotnet"));
+    if (!is_running_in_wow64 &&
+        pProcessMachine != pNativeMachine &&
+        pNativeMachine == IMAGE_FILE_MACHINE_ARM64)
```

Comment:
```suggestion
        pProcessMachine == IMAGE_FILE_MACHINE_AMD64)
```
Nit: This would be slightly more future proof.